test/helpers/kubectl.go

```diff
@@ -571,7 +571,7 @@ func (kub *Kubectl) PrepareCluster() {
 
 // labelNodes labels all Kubernetes nodes for use by the CI tests
 func (kub *Kubectl) labelNodes() error {
-	cmd := KubectlCmd + " get nodes -o json | jq -r '[ .items[].metadata.name ]'"
+	cmd := KubectlCmd + " get nodes --selector=node-role.kubernetes.io/controlplane!=,node-role.kubernetes.io/master!= -o json | jq -r '[ .items[].metadata.name ]'"
```

Comment:
I think you cannot exclude labels via selector right now, you should use jq for that.

`kubectl get nodes -o json | jq -r '[.items[] | select(.metadata.labels["node-role.kubernetes.io/controlplane"] == null).metadata.name]'`

Reply:
That selector works for me locally on a kubeadm cluster (it selects the nodes which don't have those labels at all, it ignores the nodes having either `node-role.kubernetes.io/controlplane` or `node-role.kubernetes.io/master` with value "", kubeadm never sets any non-empty value for those labels.). But the command you proposed looks safer and is not confusing like mine, so I will use it. Thanks!